src/client/components/Theme.tsx

```diff
@@ -0,0 +1,70 @@
+import { createMuiTheme, withTheme } from "@material-ui/core";
```

Comment:
This should be in a `theme.ts` file, outside `components/`. The `components/` folder describe React components specifically.

Reply:
Love this though; the next step is removing all CSS files and switching to material-ui's styling system!

Reply:
That seems a little too difficult since so many things are customized according to each page